system/Config/Services.php

```diff
@@ -153,6 +155,22 @@ public static function commands(bool $getShared = true)
         return new Commands();
     }
 
+    /**
+     * Content Security Policy
+     *
+     * @return ContentSecurityPolicy
+     */
+    public static function contentsecuritypolicy(?CSPConfig $config = null, bool $getShared = true)
```

Comment:
Would `csp()` be preferable? Or is that too obscure?

Reply:
This not being camel case makes me twitch. :) I think `csp` is fine, since the helper methods are `csp_x`.

Reply:
Changed.